homu/html/queue.html

```diff
@@ -3,6 +3,7 @@
     <head>
         <meta charset="utf-8">
         <title>Homu queue - {{repo_name}}</title>
+        <link href="//cdn.datatables.net/1.10.4/css/jquery.dataTables.min.css">
```

Comment:
I know including a stylesheet in `<link>` is recommended in the HTML standard, but wouldn't it slow down the initial load time if the CDN is down? Or, maybe invalidating the DOM due to the late CSS insertion would be more critical? What's your opinion on this?